modules/fts/pages/fts-creating-index-with-rest-api.adoc

```diff
@@ -1,42 +1,135 @@
-* Headers to specify settings for `cache-control` (`no-cache`) and `application-type` (`application/json`).
-* A body containing the JSON document that defines the index to be created.
-This must include the name of the bucket on which the index is to be created.
+The REST API can be used to create indexes via a simple authenticated PUT of a a JSON index-definition.
 
-The simplest way to create the appropriate JSON index-definition for the body is to create an index by means of the Couchbase Web Console, make a copy of the JSON index-definition produced (by accessing the xref:fts-creating-index-from-UI-classic-editor.adoc#using-the-index-definition-preview[Using the Index Definition Preview], explained above), modify the index-definition as appropriate, and finally, add the index-definition to the other, preceding elements required for the call.
+[{tabs}] 
+====
+Index Creation via REST::
++
+--
+The below `curl` example demonstrates the creation of an index named `demoIndex`, on the `inventory` scope  and `airline` collection, within the `travel-sample`.`inventory` bucket and scope.
+It is assumed that the command is run on the Couchbase Server or `localhost`, and that the required username and password are set in the environment variables `$CB_USERNAME` and `$CB_PASSWORD` for authentication.
 
-Note, however, that this requires modification of the `uuid` field; since the re-specifying of an existing field-value is interpreted as an attempted _update_, to an existing index.
-Therefore, if the `uuid` field for an existing index appears in the Index Definition Preview as `"uuid": "3402702ff3c862c0"`, it should be edited to appear `"uuid": ""`.
-A new ID will be allocated to the new index, and this ID will appear in the Index Definition Preview for the new index.
+[source,bourne]
+----
+curl -XPUT -H "Content-Type: application/json" \
+-u ${CB_USERNAME}:${CB_PASSWORD} http://localhost:8094/api/index/demoIndex -d \
+'{
+ "name": "demoIndex",
+ "type": "fulltext-index",
+ "params": {
+  "mapping": {
+   "types": {
+    "inventory.airline": {
+     "enabled": true,
+     "dynamic": true
+    }
+   },
+   "default_mapping": {
+    "enabled": false,
+    "dynamic": true
+   },
+   "default_type": "_default",
+   "default_analyzer": "standard",
+   "default_datetime_parser": "dateTimeOptional",
+   "default_field": "_all",
+   "store_dynamic": false,
+   "index_dynamic": true,
+   "docvalues_dynamic": false
+  },
+  "store": {
+   "indexType": "scorch",
+   "kvStoreName": ""
+  },
+  "doc_config": {
+   "mode": "scope.collection.type_field",
+   "type_field": "type",
+   "docid_prefix_delim": "",
+   "docid_regexp": ""
+  }
+ },
+ "sourceType": "couchbase",
+ "sourceName": "travel-sample",
+ "sourceParams": {},
+ "planParams": {
+  "maxPartitionsPerPIndex": 1024,
+  "numReplicas": 0,
+  "indexPartitions": 1
+ }
+}'
+----
+If the REST call is successful, the following object is returned:
+[source,bourne]
+----
+{"status":"ok"}
```

Comment:
This isn't accurate, a "uuid" is also returned as part of the response.

I see you've included it correctly in the section for index updation below.

Reply:
fixed, thanks